api/v1/sriovnetworknodestate_types.go

```diff
@@ -65,6 +65,7 @@ type InterfaceExt struct {
 	LinkType    string            `json:"linkType,omitempty"`
 	EswitchMode string            `json:"eSwitchMode,omitempty"`
 	TotalVfs    int               `json:"totalvfs,omitempty"`
+	NUMANode    int               `json:"numaNode,omitempty"`
```

Comment:
Why do we need to add this ? what is the use case ?

just for general information for the user, is not good enough IMO. a user may want to know alot of things about the node. perhaps this info should be exposed by node-feature-discovery ?

Reply:
if the info is exposed by nfd, the user still needs to map the info exposed by node state to the nfd though. Which might be ok, but it's important to realize this imo.

Reply:
Yes, my idea was to make it available for general information to the user, as the other NIC related properties (PCI, vendor, deviceID). It would also make testing NUMA-related features a bit easier, but I understand it can be too little for an API change.